### ISSUE TITLE
FAQ structure

### ISSUE THREAD

**Issue by ngtuna** [tomochain/docs#94]
**Estimated time:** ?h

---------------

**what:**
Update the FAQs of the docs website.
We want to structure it this way:
```markdown
General
  - ...
  - FAQ
Product (TomoScan, TomoMaster, TomoChain, Masternode, TomoX, ...)
  - ...
  - FAQ
```

**how:**
This will be a collaborative work in a few steps:
- Assigned writer and collaborators write and collect questions (shared google docs, link given via pm)
- Questions selections
- Assigned writer organize the documents and write the answers

**Comment by etienne-napoleone** [comment]
Assigned to: @DuqueKarl 
Reviewed by: @etienne-napoleone

**Comment by etienne-napoleone** [comment]
Hey @DuqueKarl any new on the advancement of the FAQ? Just pm me if you need any help

**Comment by alex9121** [comment]
Hi guys, 
Please check it out here:  https://docs.google.com/spreadsheets/d/1DJ9VzimKXG2r67POMOdmTXrDLonx2-H-yLw38sZyzGg/edit#gid=0

**Comment by DuqueKarl** [comment]
https://github.com/tomochain/docs/pull/111

**Comment by phamvancam2104** [comment]
closed in #111
